plugins/woocommerce/src/StoreApi/Schemas/V1/CartItemSchema.php

```diff
@@ -83,6 +83,69 @@ public function get_item_response( $cart_item ) {
 		];
 	}
 
+	/**
+	 * Get list of product images for the cart item.
+	 *
+	 * @param \WC_Product $product       Product instance.
+	 * @param array       $cart_item     Cart item array.
+	 * @param string      $cart_item_key Cart item key.
+	 * @return array
+	 */
+	protected function get_cart_images( \WC_Product $product, array $cart_item, string $cart_item_key ) {
+		$product_images = $this->get_images( $product );
+
+		/**
+		 * Filter the cart product images.
+		 *
+		 * This hook allows the cart item images to be changed. This is specific to the cart endpoint.
+		 *
+		 * @param array  $product_images  Array of image objects, as defined in ImageAttachmentSchema.
+		 * @param array  $cart_item      Cart item array.
+		 * @param string $cart_item_key  Cart item key.
+		 * @since 9.4.0
+		 */
+		$filtered_images = apply_filters( 'woocommerce_store_api_cart_item_images', $product_images, $cart_item, $cart_item_key );
```

Comment:
Hey @opr ! I honestly don't understand why cart item would need multiple images, I guess this is because the schema class is extending the product one and that one does support multiple images.

I was going to suggest we move this to `woocommerce_cart_item_thumbnail` but sadly that one returns html for an image.

I think we go in the future toward just a single image we can still make this filter backward compatible.

Reply:
I think so, by passing an array containing a single image